src/Infrastructure/Controller/Regulation/ExportRegulationController.php

```diff
@@ -34,14 +36,18 @@ public function __construct(
     )]
     public function __invoke(string $uuid): Response
     {
-        $regulationOrderRecord = $this->getRegulationOrderRecordUsing(function () use ($uuid) {
-            return $this->queryBus->handle(new GetRegulationOrderRecordSummaryQuery($uuid));
+        /** @var GeneralInformationView */
+        $generalInformation = $this->getRegulationOrderRecordUsing(function () use ($uuid) {
+            return $this->queryBus->handle(new GetGeneralInformationQuery($uuid));
         });
 
+        $regulationOrderLocations = $this->queryBus->handle(new GetRegulationLocationsQuery($uuid));
+
         $content = $this->twig->render(
             name: 'regulation/export.md.twig',
             context: [
-                'regulationOrderRecord' => $regulationOrderRecord,
+                'generalInformation' => $generalInformation,
+                'regulationOrderRecord' => $regulationOrderLocations,
```

Comment:
Synchroniser les variables du template

```suggestion
                'regulationOrderLocations' => $regulationOrderLocations,
```